main.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.start_console
+end
```

Comment:
- [ ] Everything is awesome with this main file and this main method 👍 but your app is not running on running ruby main.rb 😨 because you have not called this main method anywhere in this main.rb file 😄 Kindly call this method main at the end line of this main.rb file 👍